sql/src/test/java/io/crate/integrationtests/SelectEscapedCharactersITest.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Licensed to Crate under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.  Crate licenses this file
+ * to you under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied.  See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial
+ * agreement.
+ */
+
+package io.crate.integrationtests;
+
+import io.crate.testing.TestingHelpers;
+import org.elasticsearch.test.ESIntegTestCase;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.hamcrest.Matchers.is;
+
+@ESIntegTestCase.ClusterScope(numDataNodes = 1)
+public class SelectEscapedCharactersITest extends SQLTransportIntegrationTest {
```

Comment:
What exactly are we testing here with these integration tests? Seems that they're either redundant or incomplete (and on a too high level = very expensive).

If we want to test the indexing and querybility we can create a unit test using the `QueryTester`, but that would amount to testing Lucene and I'm not sure if that is what we should be doing?

Reply:
@mfussenegger yes, I was curious to test the querybiliy and make sure it works as expected, that was the reason for this integration test. I do agree though that it's too expensive and perhaps a bit redundant to keep thus I removed it. 

Did not know about `QueryTester` so thanks for that. 
Do you feel that more tests are required?